### ISSUE TITLE
Non-capitalized sentence start

### ISSUE THREAD

**Issue by AngledLuffa** [UniversalDependencies/UD_Italian-VIT#3]
I found a sentence start in the train section where the text has a capital letter, but the first word is not capitalized

```
# sent_id = VIT-6109
# text = Al di là di pregiudiziali e di steccati ideologici e con la partecipazione non solo dei partiti, ma di associazioni, movimenti, singole personalità, occorre promuovere un confronto aperto sulle grandi scelte per l'avvenire della società italiana.
1-2     Al      _       _       _       _       _       _       _       _
1       a       a       ADP     E       _       4       case    _       _
2       il      il      DET     RD      Definite=Def|Gender=Masc|Number=Sing|PronType=Art       4       det     _       _
```

**Comment by mr-martian** [comment]
In this case the token corresponding to the first word of the text is `1-2`, which is capitalized. Lines `1` and `2`, since they are part of a multiword token, do not need to correspond to the text.

**Comment by AngledLuffa** [comment]
The annotation is not consistent with the way this token is annotated
elsewhere, though.  The PR would fix that consistency

**Comment by AngledLuffa** [comment]
Merged in #5 , thank you!
